phpstan.neon.dist

```diff
@@ -25,6 +25,11 @@ parameters:
         - "~Access to an undefined property GraphQL\\\\Language\\\\AST\\\\.+::\\$didEnter~"
         - "~Access to an undefined property GraphQL\\\\Language\\\\AST\\\\.+::\\$didLeave~"
         - "~Access to an undefined property GraphQL\\\\Language\\\\AST\\\\Node::\\$value~"
+        # TODO fix those newly added errors and finish upgrading to PhpStan 0.12
```

Comment:
I think we can do those in separate PR's, since those throw over a hundred errors if you combine them.